preview/Win7Msix/Win7MSIXInstaller/IPackageHandler.hpp

```diff
@@ -9,12 +12,14 @@ class IPackageHandler
     virtual HRESULT ExecuteForAddRequest() = 0;
 
     virtual HRESULT ExecuteForRemoveRequest() { return S_OK; }
+    virtual bool IsMandatoryForRemoveRequest() { return false; }
+    virtual bool IsMandatoryForAddRequest() { return true; }

```

Comment:
Why do we need these APIs?

Reply:
For RemovePackage for example, some handlers are mandatory (PopulatePackageInfo) while others can be ignored if they return an error